benchmarks/nightly.sh

```diff
@@ -7,8 +7,9 @@ ACCELERATOR=a100
 OUTPUT_DIR=${HOME:?}
 WORKSPACE=$(date --utc +%Y-%m-%d)
 REPEAT=8
+ENABLE_PROFILING=0
 
-while getopts 'A:O:R:W:' OPTION
+while getopts 'A:O:PR:W:' OPTION
```

Comment:
To clarify, is this meant to be `'A:O:P:R:W:'`?

Reply:
I don't think so, the `-P` seems don't need any input argument.